### PR TITLE
Update pa11y-webservice to use pa11y 5.x

### PR DESCRIPTION
**Pull request by wilco42**
Implements #56 

A couple of notes:

1) This is a major version upgrade (?) mostly because pa11y 5.x requires node 8+.
2) pa11y 5.x's default behavior of not showing warnings or notices deviates from 4.x, but to keep parity with pa11y-dashboard's existing behavior, we have defaulted this to true.

### CODE REVIEW

**Comment by rowanmanning** [comment]
Thanks @wilco42, this is great. I'm gonna need a little time to review properly and test but appreciate the work you've put in :)

**Comment by hollsk** [comment]
We've just got this branch running at Springer Nature and it's looking great! 👍 

We did have to make [one change in `model/task.js`](https://github.com/springernature/pa11y-webservice/blob/603fb26bf5dd111ae7328b513761f9a42ec93b35/model/task.js#L172), so that we could set the `--no-sandbox` flag when Chrome launches. We removed the `phantom` object from that config too, as it won't be needed anymore.

Somebody else had [the same issue on StackOverflow](https://stackoverflow.com/questions/51620971/passing-options-for-headless-chrome) recently, so I think this is probably going to be fairly common and something that we should be allowing people to pass in. 

It shouldn't be the default option. I think it should be a global setting, not a task-specific setting - if one task needs it, they probably all will, right? As it's related to the environment that Chrome is running in. Does it seem right to add it as an option in the config object for webservice?

**Comment by paazmaya** [comment]
@rowanmanning any chance of getting this PR reviewed?

I am looking to work also on updating dependencies and making this run within Chrome...

**Comment by joeyciechanowicz** [comment]
Looking at Travis, we have a slight problem. Pa11y 5 hikes the minimum Node version to 8. Our build jobs are for 4, 5, & 6.

File: model/task.js [hunk 1]
```diff
@@ -163,14 +163,19 @@ module.exports = function(app, callback) {
...
 						standard: task.standard,
+						includeWarnings: true,
+						includeNotices: true,
 						timeout: (task.timeout || 30000),
 						wait: (task.wait || 0),
 						ignore: task.ignore,
 						actions: task.actions || [],
-						phantom: {},
+						chromeLaunchConfig: {
+							args: ['--no-sandbox']
```
Comment (hollsk) [inline model/task.js]: `--no-sandbox` shouldn't be the default option. The Chromium team specifically warn against using it, so the fact that we're using it in the first place to get it running is the wrong thing to do 😆 so it'd be doubly-triply wrong to force it on all users, some of whom may actually have more appropriate kernel setups.

**Comment by paazmaya** [comment]
I have made updates on Node.js versions in my PR #68, which migrates to hapi v17, and which I would continue to work on further updates but since I have not seen any interest from the maintainers, I am not sure how to proceed.

**Comment by hollsk** [comment]
Hey @paazmaya - we're grateful for the work you've put in so far! We're a small team who work on Pa11y outside of our day jobs, and you've just caught us at a really busy/bad time. We're working to get back on top of things, thanks for your patience! 😄

**Pull request merged**